lib/liquid/standardfilters.rb

```diff
@@ -296,6 +296,11 @@ def replace_first(input, string, replacement = '')
       input.to_s.sub(string.to_s, replacement.to_s)
     end
 
+    # Replace the last occurrences of a string with another
+    def replace_last(input, string, replacement = '')
```

Comment:
The replacement string default is kind of strange, considering that `remove_last` should actually be used in that case.  Perhaps we should deprecate using the default argument with `replace_first` and replace that usage with `remove_first` in https://github.com/shopify/theme-check in which case it might not even make sense to add the default argument to this new filter.

Reply:
I think the new filter should have the same as the old ones. Making `replace_first` work with no `replacement` arg and making `replace_last` not work wouldn't make sense.

But all in for deprecating the feature.

Reply:
Using replace_last with only one argument will look weird and won't make sense to a reader of that liquid code.  In the case of `replace_first`, the default argument isn't documented (https://shopify.github.io/liquid/filters/replace_first/) and may not even be getting used.  Either way, I don't want to support more of that usage.